qa/L1_lenet/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+ # Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+ #
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ #     http://www.apache.org/licenses/LICENSE-2.0
+ # 
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+
+TEST_LOG="./data.log"
+
+
+apt-get update && \
+    apt-get install -y --no-install-recommends python
```

Comment:
Is python not installed on runners? Also, I think this is python2. Is that what we want?

Reply:
You don't need to install python, it is in the runners. Plus if someone else where to use this, it would be assumed they have python already, since they are looking at a profiler for pytorch. Please remove this.

Reply:
This is not needed, remove.